packages/xod-tabtest/src/TabtestValidator.rei

```diff
@@ -0,0 +1,13 @@
+open Belt;
+
+open XodProject;
+
+/**
+ * Checks tabtest for correct pin labels and throws an error if tabtest has:
+ * - missing pin labels
+ * - redundant pin labels
+ * - duplicated pin labels
+ *
+ * Otherwise do nothing.
+ */
+let assertPinLabels: (Map.String.t(Node.id), TabData.t, string) => unit;
```

Comment:
What are we asserting? Maybe, `validatePinLabels`?

:astonished: are we making an exception a part of normal flow?! Not FP. Should be a Result of an Error with code and message.